templates/security/register_user.html

```diff
@@ -17,4 +18,4 @@ <h1>{{ _fsdomain('Register') }}</h1>
         {{ render_field(register_user_form.submit) }}
     </form>
     {% include "security/_menu.html" %}
-{% endblock %}
\ No newline at end of file
+{% endblock %}
```

Comment:
eof change

Reply:
should now have no newline at end of file